### PR TITLE
loan tranch details

### PR DESCRIPTION
**Pull request by rashmikokkalaki**

### CODE REVIEW

File: src/app/modules/loan-view/loan-view.component.html [hunk 1]
```diff
@@ -1037,8 +1037,65 @@ <h6 class="mb-0">Subvention Scheme</h6>
...
+                               <!-- &nbsp;{{loandetails.maxOutstandingLoanBalance| number}}&nbsp; -->
+                            </td>
+                         </tr>
+                         <tr>
+                            <td>
+                               <b>Loan Tranche Details</b>
+                            </td>
+                         </tr>
+                      </table>
+                      <table class="width100">
```
Comment (NVMKAVYA) [inline src/app/modules/loan-view/loan-view.component.html]: remove class width100

File: src/app/modules/loan-view/loan-view.component.html [hunk 2]
```diff
@@ -1037,8 +1037,65 @@ <h6 class="mb-0">Subvention Scheme</h6>
...
+                         <tr class="graybg" *ngIf="loanDetails.disbursementDetails.length>0">
```
Comment (NVMKAVYA) [inline src/app/modules/loan-view/loan-view.component.html]: remove css class graybg

File: src/app/modules/loan-view/loan-view.component.html [hunk 3]
```diff
@@ -1037,8 +1037,65 @@ <h6 class="mb-0">Subvention Scheme</h6>
...
+                            <th align="left">Expected disbursement on</th>
+                            <th align="left">Disbursed on</th>
+                            <th align="left">Approved Amount</th>
+                            <th align="left">Principal</th>
+                            <th align="left">&nbsp;</th>
```
Comment (NVMKAVYA) [inline src/app/modules/loan-view/loan-view.component.html]: remove align=left for all th tags

File: src/app/modules/loan-view/loan-view.component.html [hunk 4]
```diff
@@ -1037,8 +1037,65 @@ <h6 class="mb-0">Subvention Scheme</h6>
...
+                         </tr>
+                         <tr *ngFor="let disbursementDetail of loanDetails.disbursementDetails">
+                            <td>{{disbursementDetail.expectedDisbursementDate | DateFormat}}</td>
```
Comment (NVMKAVYA) [inline src/app/modules/loan-view/loan-view.component.html]: replace DateFormat with date : dateFormat

File: src/app/modules/loan-view/loan-view.component.html [hunk 5]
```diff
@@ -1037,8 +1037,65 @@ <h6 class="mb-0">Subvention Scheme</h6>
...
+                            <td>{{disbursementDetail.actualDisbursementDate | DateFormat}}</td>
```
Comment (NVMKAVYA) [inline src/app/modules/loan-view/loan-view.component.html]: replace DateFormat with date : dateFormat

File: src/app/modules/loan-view/loan-view.component.html [hunk 6]
```diff
@@ -1037,8 +1037,65 @@ <h6 class="mb-0">Subvention Scheme</h6>
...
+                            <td>{{disbursementDetail.approvedPrincipal | number}}</td>
+                            <td>{{disbursementDetail.principal | number}}</td>
+                            <td>
+                               <div *ngIf="disbursementDetail.actualDisbursementDate != null"><i
+                                  class="icon-check icon-white"></i>
```
Comment (NVMKAVYA) [inline src/app/modules/loan-view/loan-view.component.html]: Replace icon-check  with fa icon

**Review by NVMKAVYA — CHANGES_REQUESTED** [review]

File: src/app/modules/loan-view/loan-view.component.html (hunk 1 above)
Comment (rashmikokkalaki) [reply src/app/modules/loan-view/loan-view.component.html]: done

File: src/app/modules/loan-view/loan-view.component.html (hunk 2 above)
Comment (rashmikokkalaki) [reply src/app/modules/loan-view/loan-view.component.html]: done

File: src/app/modules/loan-view/loan-view.component.html (hunk 3 above)
Comment (rashmikokkalaki) [reply src/app/modules/loan-view/loan-view.component.html]: done

File: src/app/modules/loan-view/loan-view.component.html (hunk 4 above)
Comment (rashmikokkalaki) [reply src/app/modules/loan-view/loan-view.component.html]: done

File: src/app/modules/loan-view/loan-view.component.html (hunk 5 above)
Comment (rashmikokkalaki) [reply src/app/modules/loan-view/loan-view.component.html]: done

File: src/app/modules/loan-view/loan-view.component.html (hunk 6 above)
Comment (rashmikokkalaki) [reply src/app/modules/loan-view/loan-view.component.html]: done

**Review by NVMKAVYA — APPROVED** [review]

**Pull request merged**